cmd/metal-api/internal/service/v1/s3.go

```diff
@@ -0,0 +1,53 @@
+package v1
```

Comment:
Currently we only store request and response definitions in here. I think you can just move it to a separate s3 package.

At some point in time we should probably move the v1 package away from `internal` to `pkg/api` or something... but that can be another PR in the future.